components/form-token-field/suggestions-list.js

```diff
@@ -81,25 +81,36 @@ class SuggestionsList extends Component {
 		// why, since usually a div isn't focusable by default
 		// TODO does this still apply now that it's a <ul> and not a <div>?
 		return (
-			<ul ref={ this.bindList } className={ classes } tabIndex="-1">
+			<ul
+				ref={ this.bindList }
+				className={ classes }
```

Comment:
This element renders with an `is-expanded` modifier class. Should it also apply `aria-expanded`? Or is this the role of the input? I'm not really sure.